models/client/reducers.js

```diff
@@ -0,0 +1,48 @@
+module.exports = globalConfig => ({
+  updateUsername: updateUsername,
+  connectivityChange: connectivityChange(globalConfig),
+  suggestRecovery: suggestRecovery,
+  denyRecovery: denyRecovery,
+  activateRecovery: activateRecovery(globalConfig)
+})
+
+function connectivityChange (globalConfig) {
+  return inner
+  function inner (state, info) {
+    state.group = info.GID
+    state.id = info.CID
+    state.connectivityState = info.state
+    return state
+  }
+}
+
+function updateUsername (state, name) {
+  state.username = name
+  return state
+}
+
+function activateRecovery(globalConfig) {
+  return inner
+  function inner(state, _) {
```

Comment:
Hey Tim, I saw the lodash at Per Guth's repository and was wondering what it does too, then i figured it basically defines a variable passed to the function which is not used. In this particular case it is not really necessary, but let's say we have an effect function with (state, data, send, done). If we do not need one of the arguments we could lodash it, thus clarifying we are not using it. Seemed neat to me so i used it ;)